src/store/growthbook-context.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { GrowthBook, GrowthBookProvider } from '@growthbook/growthbook-react'
+import { getClientInformation, getDomain, getLanguage } from 'common/utility'
+import { growthbook_client_key } from 'common/constants'
+
+let growthbook
+if (typeof window !== 'undefined') {
```

Comment:
Would it be possible to do this in a `useEffect` inside the `DerivGrowthBookProvider` instead of declaring a global variable? 🤔

Reply:
Yeah nice one. Fixed in 9adf34a6426a604876daf93ef263acb216c85c83.